### PR TITLE
Report mutator and gc time in MempoolReproForge

### PR DESCRIPTION
**Pull request by jasagredo**

### CODE REVIEW

File: ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Analysis.hs [hunk 1]
```diff
@@ -178,15 +179,19 @@ data TraceEvent blk =
...
-    -- ^ triggered for all blocks during ShowBlockTxsSize analysis,
+  | BlockMempoolAndForgeRepro BlockNo SlotNo Int SizeInBytes IOLike.DiffTime Int64 Int64 IOLike.DiffTime Int64 Int64
+    -- ^ triggered for all blocks during MempoolAndForgeRepro analysis,
     --   it holds:
     --   * block number
     --   * slot number when the block was forged
     --   * number of transactions in the block
     --   * total size of transactions in the block
     --   * time to tick ledger state
+    --   * mutator time to tick ledger state
```
Comment (dnadales) [inline ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Analysis.hs:190]: Perhaps we should say

```
Total time spent in the mutator when ticking the ledger state
```

https://github.com/input-output-hk/ouroboros-consensus/pull/400/files#diff-fa79e075b3c0bc1a549ac7ddfe0b97fc5817a24a8de7577988d15d92019316efR161

File: ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Analysis.hs [hunk 2]
```diff
@@ -178,15 +179,19 @@ data TraceEvent blk =
...
+    --   * gc time to tick ledger state
     --   * time to call 'Mempool.getSnapshotFor'
+    --   * mutator time during 'Mempool.getSnapshotFor'
```
Comment (dnadales) [inline ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Analysis.hs:193]: Same here.

**Review by dnadales — COMMENTED** [review]
Should we update the `db-analyser`'s README as well?

File: ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Analysis.hs (hunk 1 above)
Comment (jasagredo) [reply ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Analysis.hs]: Done

File: ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Analysis.hs (hunk 2 above)
Comment (jasagredo) [reply ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Analysis.hs]: Done

**Comment by jasagredo** [comment]
I updated the readme as well @dnadales

**Review by dnadales — APPROVED** [review]

**Pull request merged**